.github/renovate.json

```
@@ -1,5 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>1024pix/renovate-config:aggressive"],
-  "enabled": true
+  "extends": [
+    "github>1024pix/renovate-config:aggressive"
+  ],
+  "enabled": false
```

```suggestion
  "extends": [
    "github>1024pix/renovate-config:default"
  ],
  "enabled": true
```
Est ce qu'on peut plutôt juste mettre le mode par défaut sans automerge ?

---

ça me va :)